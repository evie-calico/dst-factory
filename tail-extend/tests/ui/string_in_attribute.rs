use tail_extend::make_dst_builder;

#[make_dst_builder("A String Literal")]
struct StringInAttribute {
    id: i32,
    data: str,
}

fn main() {
}
