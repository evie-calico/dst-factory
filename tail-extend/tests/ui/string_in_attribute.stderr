error: Unexpected input
 --> tests/ui/string_in_attribute.rs:3:20
  |
3 | #[make_dst_builder("A String Literal")]
  |                    ^^^^^^^^^^^^^^^^^^
